### DISCUSSION TITLE
Problems retrieving latency from shard.

### DISCUSSION THREAD

**Discussion by tk-nguyen** [serenity-rs/serenity#1534] [Q&A] [answered]
Hello! I'm trying to get the latency from the shard to send as a reply to the `ping` command. I've been replicating the example at https://github.com/serenity-rs/serenity/blob/current/examples/e05_command_framework/src/main.rs#L464, but `runner.latency` returns `None` every time. What should I do to resolve this? Any help would be greatly appreciated!

**Comment by kangalio** [comment, marked as answer]
Have you tried waiting a few minutes after bot startup? If I recall correctly, the shard manager builds up the latency information with time, after some traffic has been exchanged.

By the way, I'd recommend creating an issue instead for things like this in the future. It's what serenity has always been using and it's more convenient

**Reply by tk-nguyen** [reply]
Yeah, it must've been the wait. Thank you! I didn't create an issue because I don't think this is a bug in the first place...
